client/signup/steps/design-picker/index.jsx

```diff
@@ -64,16 +64,15 @@ export default function DesignPickerStep( props ) {
 			: 'free';
 
 	// Limit themes to those that support the Site editor, if site is fse eligible
-	const { isLoading: blockEditorSettingsAreLoading, data: blockEditorSettings } =
-		useBlockEditorSettingsQuery( siteId, userLoggedIn && ! props.useDIFMThemes );
-	const isFSEEligible = blockEditorSettings?.is_fse_eligible ?? false;
+	const { isLoading: blockEditorSettingsAreLoading } = useBlockEditorSettingsQuery(
+		siteId,
+		userLoggedIn && ! props.useDIFMThemes
+	);
```

Comment:
Do we care about if the block editor settings are loading (`blockEditorSettingsAreLoading`) if we aren't using them anymore in this context?

Reply:
Since the `blockEditorSettingsAreLoading` data query is no longer used, we do not care about whether it loaded the data or not. I will remove the data query and any usage of `isLoading` as well.